docs/advanced-guide/serving-static-files/page.md

```diff
@@ -0,0 +1,76 @@
+# Serving Static Files using GoFr
+
+Often times we require to serve static content be it a default profile image or a static website. We want to have a mechanism to serve those content without having a hassel of implementing it from scratch.
+
+GoFr provides a default mechanism where if a public folder is available in the directory of the application, it automatically provides an endpoint with `/public/<filename>`, here filename refers to the file we want to get static content to be served. 
+
+Example Project folder utilizing public endpoint:
+
+```
+project_folder
+|
+|---config
+|       .env
+|---public
+|       <img1>.jpeg
+|       <img2>.png
+|       <img3>.jpeg
+|   main.go
+|   main_test.go
+```
+
+main.go code:
+
+```go
+package main
+
+import "gofr.dev/pkg/gofr"
+
+func main(){
+    app := gofr.New()
+    app.Run()
+}
+
+```
+
+Additionally if we want to serve additional static endpoints, we have a open function called `AddStaticFiles()` which takes 2 parameters endpoint and the filepath of the static folder which we want to serve.
```

Comment:
`Additionally, if we` missing comma

`a open` => `an open`

Stylistic remarks:
Additionally followed by additional in same sentence seems odd
What is a open function? dedicated maybe?

Reply:
Yeah I should have rephrased it to a dedicated. Will make the changes.